week2/homework/src/commands.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable indent */
+const fs = require('fs');
+const chalk = require('chalk');
+let todoList = [];
+
+const list = () => {
+  todoList = JSON.parse(fs.readFileSync('todoList.json'));
+  if (todoList.length === 0) {
+    console.log(`There is nothing to show on your to-do list, please add some tasks`);
+  }
+ else {
+    console.log(todoList);
+  }
+};
+
+const add = todo => {
+  try {
+    todoList = JSON.parse(fs.readFileSync('todoList.json'));
+    console.log(`${todo} is added to your todo list`);
+    todoList.push({ todo });
+
+    fs.writeFileSync('todoList.json', JSON.stringify(todoList));
+  }
+ catch (err) {
+    throw Error(err);
+  }
+};
+
+const remove = index => {
+  todoList = JSON.parse(fs.readFileSync('todoList.json'));
+  todoList.splice(index - 1, 1);
+  console.log(`The item in the ${index} place has been removed`);
+  fs.writeFileSync('todoList.json', JSON.stringify(todoList));
+};
+
+const update = (index, str) => {
+  todoList = JSON.parse(fs.readFileSync('todoList.json'));
+  todoList.splice(index - 1, 1, { str });
+  console.log(`The item in the ${index} place is up-to date`);
+  fs.writeFileSync('todoList.json', JSON.stringify(todoList));
+};
+
+const reset = () => {
+  try {
+    todoList = JSON.parse(fs.readFileSync('todoList.json'));
+    todoList.length = 0;
+    fs.writeFileSync('todoList.json', JSON.stringify(todoList));

```

Comment:
The only thing this function does is to rewrite the file making it an empty array, that would be great to have but on the other hand when we are removing with no care of what content is, then there is no need for the read on line 45 and line 46.

the solution will be a super easy one after removing lines 45, 46 we could do:
```suggestion
    fs.writeFileSync('todoList.json', JSON.stringify([])); // done :)
```